packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -28,13 +28,16 @@ int main_( Teuchos::CommandLineProcessor &clp, int argc, char *argv[] )
     int n_values = 50000;
     int n_queries = 20000;
     int n_neighbors = 10;
+    int buffer_size = 5;
     bool perform_knn_search = true;
     bool perform_radius_search = true;
 
     clp.setOption( "values", &n_values, "number of indexable values (source)" );
     clp.setOption( "queries", &n_queries, "number of queries (target)" );
     clp.setOption( "neighbors", &n_neighbors,
                    "desired number of results per query" );
+    clp.setOption( "buffer", &buffer_size,
+                   "size for buffer optimization in radius search" );
```

Comment:
I think that the name `buffer_size` is misleading because it's not the size of the buffer. It's only related to it.